index.js

```diff
@@ -68,11 +68,12 @@ class Select2 extends Component {
     }
 
 
-    handleConfirm() {
+    handleConfirm(recivedSelectedItem = []) {
```

Comment:
E se quem chamar o `handleConfirm` passar uma lista vazia pq não tem nada selecionado? Talvez seria melhor deixar o parametro undefined mesmo, assim fica claro que quem chamou essa função não está passando esse argumento
```suggestion
    handleConfirm(recivedSelectedItem) {
```